legal-api/src/legal_api/services/filings/validations/change_of_name.py

```diff
@@ -13,26 +13,55 @@
 # limitations under the License.
 """Validation for the Change of Name filing."""
 from http import HTTPStatus
-from typing import Dict
+from typing import Dict, Final
 
-from flask_babel import _
+from flask_babel import _ as babel  # noqa: N81
 
 from legal_api.errors import Error
 from legal_api.models import Business
+from legal_api.services import namex
 
 from ...utils import get_str
 
 
 def validate(business: Business, con: Dict) -> Error:
     """Validate the Change of Name filing."""
     if not business or not con:
-        return Error(HTTPStatus.BAD_REQUEST, [{'error': _('A valid business and filing are required.')}])
+        return Error(HTTPStatus.BAD_REQUEST, [{'error': babel('A valid business and filing are required.')}])
     msg = []
     legal_name_path = '/filing/changeOfName/legalName'
     legal_name = get_str(con, legal_name_path)
     if not legal_name:
-        msg.append({'error': _('Legal Name must be provided.'),
+        msg.append({'error': babel('Legal Name must be provided.'),
                     'path': legal_name_path})
     if msg:
         return Error(HTTPStatus.BAD_REQUEST, msg)
     return None
+
+
+def validate_v2(business: Business, filing: Dict) -> Error:
```

Comment:
can we update the existing change of name validation tests to test this new validation function?